components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -88,6 +89,13 @@ open class GleanInternalAPI internal constructor () {
         applicationContext: Context,
         configuration: Configuration = Configuration()
     ) {
+        // In certain situations Glean.initialize may be called from a process other than the main
+        // process.  In this case we want initialize to be a no-op and just return.
+        if (!applicationContext.isMainProcess()) {
+            logger.error("Attempted to initialize Glean on a process other than the main process")
```

Comment:
@travis79 can you make sure that it's ok to call the `Logger` on other processes?

Reply:
After experimenting with this, it appears to log as intended.